### ISSUE TITLE
Baselining: revert to minimum necessary version number 

### ISSUE THREAD

**Issue by nport** [bndtools/bndtools#1126]
Eclipse platform: Windows 7, Bndtools version 2.4.1

These steps shows you an interesting feature that could be useful to have:

1. Enable baselining on your api project and "release" the current bundle version (that is, for instance 1.0.1.
2. Issue a modification on the sourcecode that requires some version updates, according to the semantic versioning rules.
3. Update the versions, according to the suggestions provided by bndtools (for instance, change the bundle version to 2.0.0).
4. Revert now the modification done (you changed your mind).

It would be great if bndtools tells you that the current version is not aligned anymore with the semantic versioning rules. For instance it could simply show you a warning similar to: "A correct version number for your bundle/package would be 1.0.1 instead of 2.0.0".

In other words, it seems to me, that in the current implementation, baselining takes places if the version is too low, and not in case it is too high.
